api/test/entity-mocks.ts

```diff
@@ -280,6 +281,17 @@ async function createSourcingRecord(
   return sourcingRecord.save();
 }
 
+async function createTask(status: TASK_STATUS) {
+  const task: Task = new Task();
+  task.createdBy = '2a833cc7-5a6f-492d-9a60-0d6d056923ea';
+  task.type = TASK_TYPE.SOURCING_DATA_IMPORT;
+  task.status = status;
+  task.data = {
+    filename: 'fakeFile.xlsx',
+  };
+  return task.save();
+}
```

Comment:
Plese could you add the ability to add additional / overwritte default data? Have a look how it is done with the rest of the entities in the mock utility, it has proven to be super useful more than once :)

Reply:
thanks a lot, all fixed :)